src/js/components/Meter/Circle.js

```diff
@@ -48,10 +54,46 @@ const Circle = forwardRef((props, ref) => {
           onMouseLeave: () => onHover(false),
         };
       }
-      const stroke = strokeProps(
+
+      let stroke = strokeProps(
         someHighlight && !highlight ? background : colorName,
         theme,
       );
+
+      const useGradient = color && Array.isArray(color);
+
+      if (useGradient) {
```

Comment:
```suggestion
      if (color && Array.isArray(color)) {
```
Since we only use this variable one I think we don't have to explicitly define it